### ISSUE TITLE
Difference between a wallet and an account

### ISSUE THREAD

**Issue by jeevan-shikaram** [ethereum/wiki#407]
Hi,

As the title says, what's the difference between a wallet and an account?

Can I send ETH directly to an account without a wallet?

Thanks!

A quick explanation on contracts would be helpful as well.

**Comment by Alexila** [comment]
Account you have with the bank, broker, insurance company, night club. They open it for you. One day they don't like you, they close your "account". Before you deposit money to bank "account", money is yours. Once money is in the "account", it's "theirs". What you see in your bank account are only numbers, IOU. 

Wallet is just like your pocket wallet. It's yours and no one can close it, but you. 
>Can I send ETH directly to an account without a wallet?
You send ETH (or other cryptocurrency) to an ADDRESS, not account. Address is associated with the wallet. Wallet can have multiple addresses. Yes, you can send ETH without the wallet, e.g. from exchanges.

**Comment by jeevan-shikaram** [comment]
Thanks for the explanation!

So I transferred some ETH from my Jaxx wallet and the transactions haven't shown up into my account.

[First Account](https://etherscan.io/tx/0x4e165b029a74221f087e76de2ea32682d75fb78b0c6e2e3a21806eddb630c86d)

[Second Account](https://etherscan.io/tx/0x4d5ef79a747a2b1d0f80b3ab6cdc661b11e570909334c1364b0ae00ce6bfdbab)

I think it's because I haven't been synced into the block chain, but when I tried making a wallet contract it failed because I didn't have enough ETH or gas. 

How do I get around this and is the fact that I don't have a wallet contract yet affecting my ability to receive funds? And who currently "owns" these "accounts"?
